lib/devices/android/android-common.js

```diff
@@ -12,7 +12,8 @@ var logger = require('../../server/logger.js').get('appium')
   , Device = require('../device.js')
   , NotYetImplementedError = errors.NotYetImplementedError
   , Args = require("vargs").Constructor
-  , ADB = require('./adb.js');
+  , ADB = require('./adb.js')
+  , temp = require('temp');
```

Comment:
instead of this, can you use the tempDir support from the `appium-support` package? we're trying to standardize on that